src/pages/privacy-policy.js

```diff
@@ -3,20 +3,23 @@ import Layout from '../components/layout/Layout';
 
 const Index = () => (
   <Layout>
```

Comment:
In addition to changing `section` => `article`, I made some minor changes to grammar, punctuation, and content.